app/controllers/admin/project_groups_controller.rb

```diff
@@ -54,6 +54,7 @@ def destroy
 
   def render_modal_partial(status: 200)
     link_params = params.slice(:project_id, :parent_id)
+    @parents = @project.timeline_groups_preordered.to_a - [@entry] - @entry.descendants.to_a
```

Comment:
I think you can use `@entry.self_and_descendants.to_a` instead of `[@entry] - @entry.descendants.to_a`.